beacon-chain/core/types/attestation.go

```diff
@@ -119,42 +140,22 @@ func (a *Attestation) AggregateSig() []uint64 {
 // VerifyProposerAttestation verifies the proposer's attestation of the block.
 // Proposers broadcast the attestation along with the block to its peers.
 func (a *Attestation) VerifyProposerAttestation(pubKey [32]byte, proposerShardID uint64) error {
-
 	// Verify the attestation attached with block response.
 	// Get proposer index and shardID.
-
 	attestationMsg := AttestationMsg(
 		a.ObliqueParentHashes(),
 		a.ShardBlockHash(),
 		a.SlotNumber(),
 		proposerShardID,
 		a.JustifiedSlotNumber(),
-		params.BeaconConfig().InitialForkVersion)
-
-	log.Debugf("Constructing attestation message for incoming block %#x", attestationMsg)
-
+		params.BeaconConfig().InitialForkVersion,
+	)
+	_ = attestationMsg
+	_ = pubKey
```

Comment:
Delete?

Reply:
If I delete this we'll get unused var from gometalinter

Reply:
OK, and I assume that you will definitely need it in the future then?